### PR TITLE
Shadow luminosity refactoring

### PR DESCRIPTION
**Pull request by jorgeamadosoria**
changed the value of --dm-color-shadow to allow for it to be seen better in dark color scheme.

also, added a small margin to menu items to make them show properly when selected via tabbing.

### CODE REVIEW

**Review by ericcornelissen — APPROVED** [review]
> changed the value of `--dm-color-shadow` to allow for it to be seen better in dark color scheme.

This change looks good to me, thanks for fixing :+1: 

> also, added a small margin to menu items to make them show properly when selected via tabbing.

Can you elaborate on this change? While I'm okay with the change, I'm not seeing how this "makes them show properly when selected via tabbing" :confused:

**Comment by jorgeamadosoria** [comment]
In chrome, there is a thin blue outline around the menu items that shows when tabbing only.
It doesn't affect functionally but it looks as if the menu is "crowded"

**Comment by ericcornelissen** [comment]
> In chrome, there is a thin blue outline around the menu items that shows when tabbing only.
It doesn't affect functionally but it looks as if the menu is "crowded"

I see, you could alternatively use `outline: none` as is done on other elements. (I hadn't noticed the default outline on these elements before as the one in Firefox is much less pronounced)

**Comment by jorgeamadosoria** [comment]
> > In chrome, there is a thin blue outline around the menu items that shows when tabbing only.
> > It doesn't affect functionally but it looks as if the menu is "crowded"
> 
> I see, you could alternatively use `outline: none` as is done on other elements. (I hadn't noticed the default outline on these elements before as the one in Firefox is much less pronounced)

changed margin for outline:none.

File: public/stylesheet.css [hunk 1]
```diff
@@ -229,7 +229,7 @@ a {
   }
 }
 @media (max-width: 640px) {
-  .header {
+  x .header {
```
Comment (ericcornelissen) [inline public/stylesheet.css:232]: This doens't look right @jorgeamadosoria

Comment (jorgeamadosoria) [reply public/stylesheet.css:232]: fixed in #135

**Pull request merged**